rankings/views.py

```diff
@@ -2,14 +2,15 @@
 
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.messages.views import SuccessMessageMixin
+from django.core.exceptions import ObjectDoesNotExist
```

Comment:
You don't need to import this. You can target the `Object.DoesNotExist` in your exception and be more specific. See my note below.